src/wallet/rpcwallet.cpp

```diff
@@ -492,10 +492,10 @@ UniValue CreateColdStakeDelegation(const UniValue& params, CWalletTx& wtxNew, CR
         fForceNotEnabled = params[5].get_bool();
 
     int nBestHeight = chainActive.Height();
-    if (!Params().Cold_Staking_Enabled(nBestHeight) && !fForceNotEnabled) {
-        std::string errMsg = strprintf("Cold Staking not enforced yet at block %d.\n"
-                "If the wallet is syncing, you may force the stake delegation setting fForceNotEnabled to true.\n"
-                "WARNING: If the network hasn't reached the activation height, this tx will be rejected resulting in a ban.\n", nBestHeight);
+    if (!IsSporkActive(SPORK_17_COLDSTAKING_ENFORCEMENT) && !fForceNotEnabled) {
+        std::string errMsg = strprintf("Cold Staking disabled %d.\n"
```

Comment:
May want to remove the current height from the error message; or put enough in the message so the user will know what the random number in the message is.  Original design stated "at block", so it was clear that was a block number; but the current message may be confusing.

Reply:
Yep, nice catch. Unspotted remnant from previous (height-based) activation. We don't need that at all.